cli/tsc/dts/lib.deno.shared_globals.d.ts

```diff
@@ -459,6 +459,19 @@ interface DOMStringList {
 type BufferSource = ArrayBufferView | ArrayBuffer;
 
 /** @category I/O */
+/**
```

Comment:
This looks fishy - I think `@category` needs to be moved into this JS doc

Reply:
ah good spot, updated